integrated_channels/moodle/migrations/0028_auto_20231116_1826.py

```diff
@@ -6,7 +6,7 @@
 class Migration(migrations.Migration):
```

Comment:
As per discussion on Slack, we can keep migrations as is. So please remove this change.

Reply:
@sameenfatima78 updated.